pkg/core/bootstrap/autoconfig.go

```diff
@@ -57,9 +69,13 @@ func autoconfigureSds(cfg *kuma_cp.Config) error {
 	// to improve UX, we want to auto-generate TLS cert for SDS if possible
 	if cfg.Environment == config_core.UniversalEnvironment {
 		if cfg.SdsServer.TlsCertFile == "" {
+			sdsHost := strings.Split(cfg.ApiServer.Catalog.Sds.HostPort, ":")[0]
 			hosts := []string{
-				cfg.BootstrapServer.Params.XdsHost,
 				"localhost",
+				cfg.BootstrapServer.Params.XdsHost,
```

Comment:
Why are we adding Xds hostname to the SDS Server cert?

Reply:
As XDS is serving TLS so i added xds host to the cert. This created cert can be added to config when kuma-cp starts. It can be removed too. please suggest.

Reply:
Please remove it. If you need such a shared certificate you can generate it yourself pretty easily with `kumactl generate tls-certificate` providing multiple hostnames.